src/index.ts

```diff
@@ -625,18 +626,46 @@ async function listNeurons(showZeroStake: boolean = false) {
       version: "2.0.0",
     }),
   });
+  const ledger = LedgerCanister.create({
+    agent: await getCurrentAgent(new AnonymousIdentity()),
+    hardwareWallet: true,
+  });
 
   // We filter neurons with no ICP, as they'll be garbage collected by the governance canister.
   const neurons = await governance.listNeurons({
     certified: true,
   });
 
   if (neurons.length > 0) {
-    neurons
-      .filter((n) => showZeroStake || hasValidStake(n))
-      .forEach((n) => {
-        log(`Neuron ID: ${n.neuronId}`);
-      });
+    // Render a table with the neurons.
+    let idx = 0;
+    const tableData = await Promise.all(
+      neurons
+        .filter((n) => showZeroStake || hasValidStake(n))
+        .map(async (n) => {
+          const accountIdentifier = AccountIdentifier.fromHex(
+            n.fullNeuron!.accountIdentifier
+          );
+
+          const balance = await ledger.accountBalance({
```

Comment:
Why don't you use `cachedNeuronStake - neuronFees` from the [Neuron](https://github.com/dfinity/ic-js/blob/85b321dc2b66d1283c8e14306d8c6daa8a7c29cc/packages/nns/src/types/governance_converters.ts#L248)?

This way you'd avoid an extra call per neuron.

Reply:
Because these are not necessarily accurate - there's a delay between that cached number and the real value, so we might as well get the real value. The calls to the ledger are quite cheap.

Reply:
But a `query` call is not secure. Is that still better than the information in the neuron?

Or should we do an update call `certified: true`?